floss/language/identify.py

```diff
@@ -117,6 +130,19 @@ def is_go_bin(pe: pefile.PE) -> bool:
                     # just for testing
                     logger.info("Go binary found with version %s", get_go_version(magic))
                     return True
+
+    # if not found, the magic bytes are patched, search for common Go functions present in all Go samples incl. obfuscated
+    for go_function in go_functions:
+        for section in pe.sections:
```

Comment:
- can we limit to specific sections?
- would searching in the sections first for all strings be more efficient?

Reply:
agree with #2 here - pefile.get_data is fairly slow, so recommend looping across sections in the outer loop, and then across the strings in the inner loop.

Reply:
I suggest searching first in .rdata, where the string runtime.main is found